docs/sources/examples/mongodb.md

```diff
@@ -133,19 +136,19 @@ as daemon process(es).
 
     # Basic way
     # Usage: docker run --name <name for container> -d <user-name>/<repository>
-    $ docker run --name mongo_instance_001 -d my/repo
+    $ docker run -p 27017:27017 --name mongo_instance_001 -d my/repo
 
     # Dockerized MongoDB, lean and mean!
     # Usage: docker run --name <name for container> -d <user-name>/<repository> --noprealloc --smallfiles
-    $ docker run --name mongo_instance_001 -d my/repo --noprealloc --smallfiles
+    $ docker run -p 27017:27017 --name mongo_instance_001 -d my/repo --noprealloc --smallfiles
 
     # Checking out the logs of a MongoDB container
     # Usage: docker logs <name for container>
     $ docker logs mongo_instance_001
 
     # Playing with MongoDB
-    # Usage: mongo --port <port you get from `docker ps`> 
-    $ mongo --port 12345
+    # Usage: mongo --port <port you get from `docker ps`>  --host <ip address from `boot2docker ip`>
+    $ mongo --port 27017 --host 192.168.59.103
```

Comment:
A developer will have either the MongoDB client, a GUI (e.g. MongoVU) or their application connecting to the running MongoDB instance. Making the *assumption* there is a client to connect with does not seem a huge stretch and would avoid making this seem overly complex.